doc/theory/glossary.tex

```diff
@@ -307,76 +304,69 @@
 	\begin{equation*}
 	\sup_{x \in D_1 \cup \Delta} ||x - \gamma(x)||_{\infty.}
 	\end{equation*}
+
+	The set of persistence diagrams together with any of the distances above is a
+	%\hyperref[metric space]{metric space}.
```

Comment:
Even if this is commented out, I recommend using the correct label, i.e. "metric_space", inside the square brackets.

Reply:
This seems still unaddressed, or am I missing something?

Reply:
Sorry :-(

Reply:
Still not showing up here :/